src/NUnitTestAdapter/Properties/AssemblyInfo.cs

```diff
@@ -12,7 +12,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("NUnit Software")]
 [assembly: AssemblyProduct("NUnit.VisualStudio.TestAdapter")]
```

Comment:
Yes, now that we have made the decision to do a separate adapter for v3, I'll need to do another commit with all those changes.